.github/workflows/build-artifact.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
       - uses: ./.github/actions/install-python-deps
       - name: Build tt-metal and libs
         run: |
-          cmake -B build -G Ninja
+          cmake -B build -G Ninja -DCMAKE_CXX_COMPILER=clang++-17
```

Comment:
Any plans to make this default? @TT-billteng

Reply:
I don't want to force the user to be using clang++17. If/when we move to ubuntu 24.04, gcc 13 may give better perf numbers. So at best I'd like to "recommend" the compiler of choice as of now